client/components/LearnerViewDirective.js

```diff
@@ -151,6 +147,14 @@ tie.directive('learnerView', [function() {
           $scope.$apply();
         };
 
+        $scope.codeMirrorOptions = {
+          autofocus: true,
+          lineNumbers: true,
+          mode: LANGUAGE_PYTHON,
```

Comment:
I lean against, since that would be a major product decision which I doubt would slip through the cracks anyway. We'd also have to add a similar TODO in many more places, like the preprocessor and evaluator. So, leaving as-is.